frontend/src/Header/UserControls.js

```diff
@@ -0,0 +1,29 @@
+import React, { Component } from 'react';
+import { Link } from 'react-router-dom';
+import { withRouter } from 'react-router-dom';
+import auth from '../services/gqlAuth'
+
+import cogSvg from '../images/cog.svg';
+import './UserControls.css'
+
+const UserControls = withRouter(({ history }) => (
+  auth.isAuthenticated() ? (
+    <div className="UserControls">
+      <img src={cogSvg} alt="Link User Settings" className="Header__settings-icon" />
+      <Link to="#">Matthew Porcher</Link>
```

Comment:
do we have a story for making this link work in the backlog yet? If not we should make one.

Reply:
I'll add it to the backlog.